File: infra/src/main/java/com/beat/infra/notification/slack/vo/block/DividerBlock.java
package com.beat.infra.notification.slack.vo.block;

import static com.beat.infra.notification.slack.vo.SlackConstant.BLOCK_TYPE_DIVIDER;

public record DividerBlock(
	String type
) implements Block {

	public static DividerBlock newInstance() {
		return new DividerBlock(BLOCK_TYPE_DIVIDER);
	}
}

File: core/infra/src/main/kotlin/com/beat/infra/external/notification/slack/vo/block/DividerBlock.kt
package com.beat.infra.external.notification.slack.vo.block

import com.beat.infra.external.notification.slack.vo.SlackConstant.BLOCK_TYPE_DIVIDER

internal data class DividerBlock(
    override val type: String,
) : Block {
    companion object {
        fun newInstance(): DividerBlock = DividerBlock(BLOCK_TYPE_DIVIDER)
    }
}
